src/components/Footer.tsx

```diff
@@ -0,0 +1,36 @@
+'use client';
```

Comment:
[Suggested] 'use client' 目前專案中都有空一行
```suggestion
'use client';

```

Reply:
✅ 已在 2024/6/01 19:43 的 commit : a963717 修改完畢